build.assets/Makefile

```diff
@@ -9,6 +9,7 @@ SRCDIR=/go/src/github.com/gravitational/teleport
 GOMODCACHE ?= /tmp/gomodcache
 DOCKERFLAGS := --rm=true -v "$$(pwd)/../":$(SRCDIR) -v /tmp:/tmp -w $(SRCDIR) -h $(HOSTNAME) -e GOMODCACHE=$(GOMODCACHE)
 ADDFLAGS ?=
+ADD_LDFLAGS = -L/opt/lib
```

Comment:
Since `ADD_LDFLAGS` is already sandwiched between `-Wl,-Bstatic` and `-Wl,-Bdynamic` it's easier to just:
```
ADD_LDFLAGS = "-lelf -lz"
```
Although I would rename the variable to be more specific, `ADD_STATIC_LIBS` or something similar.

This will allow you to avoid copying the libs to /opt.

Reply:
With this you can probably just add `-lelf -lz` to `CGOFLAGS` in the root `Makefile` and avoid all the changes in `build.assets/Makefile`